src/main/java/org/folio/consortia/domain/entity/dto/PermissionUser.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.consortia.domain.entity.dto;
```

Comment:
@SinghAdes it's still needs to be fixed, just `org.folio.consortia.domain.dto`

Reply:
fixed